rootex/framework/components/audio_component.cpp

```diff
@@ -4,3 +4,25 @@ AudioComponent::AudioComponent(bool playOnStart)
     : m_IsPlayOnStart(playOnStart)
 {
 }
+
+bool AudioComponent::setup()
+{
+	m_TransformComponent = m_Owner->getComponent<TransformComponent>().get();
+	getAudioSource()->setPosition(m_TransformComponent->getAbsoluteTransform().Translation());
+	getAudioSource()->setModel();
```

Comment:
These values should set in the constructor through Create and CreateDefault functions